ext/node/ops/os/cpus.rs

```diff
@@ -0,0 +1,301 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+use deno_core::serde::Serialize;
+
+#[derive(Debug, Default, Serialize, Clone)]
+pub struct CpuTimes {
+  pub user: u64,
+  pub nice: u64,
+  pub sys: u64,
+  pub idle: u64,
+  pub irq: u64,
+}
+
+#[derive(Debug, Default, Serialize, Clone)]
+pub struct CpuInfo {
+  pub model: String,
+  pub speed: u64,
```

Comment:
add a comment or rename variable to specify units

Reply:
added doc comment above. can't rename because its part of the public Node API https://nodejs.org/api/os.html#oscpus